server/hedley/modules/custom/hedley_municipality/plugins/content_types/topics_list/topics_list.inc

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @file
+ * Plugin definition.
+ */
+
+$plugin = [
+  'title' => t('Topics list'),
+  'description' => t("The municipality's topics displayed in colored buttons"),
+  'category' => t('Hedley'),
+  'hook theme' => 'hedley_municipality_topics_list_content_type_theme',
+  'required context' => new ctools_context_required(t('Municipality'), 'node'),
+];
+
+/**
+ * Render callback.
+ */
+function hedley_municipality_topics_list_content_type_render($subtype, $conf, $args, $context) {
+  global $language;
+  if (empty($context->data->nid)) {
+    return NULL;
+  }
+
+  // Load topics.
+  $municipality_node = $context->data;
+  $vocabulary = taxonomy_vocabulary_machine_name_load("topics_$municipality_node->nid");
+  $terms = taxonomy_get_tree($vocabulary->vid);
+
+  $topics = [];
+  foreach ($terms as $term) {
+    $wrapper = entity_metadata_wrapper('taxonomy_term', $term->tid);
+    $wrapper->language($language->language);
+    // Ignore any term that isn't translated to the current language. In
+    // such case, the term's label would be empty.
+    if (!$wrapper->label()) {
+      continue;
+    }
+
+    $topics[] = [
+      'title' => $wrapper->label(),
```

Comment:
Nope, you are missing the bigger problem here. Hint, it's starts with a capital `X` 😜